ipa-core/src/cli/playbook/mod.rs

```diff
@@ -81,7 +82,13 @@ where
 ///
 /// ## Panics
 /// If results don't match.
-pub fn validate_dp(expected: Vec<u32>, actual: Vec<u32>, epsilon: f64, per_user_credit_cap: u32) {
+pub fn validate_dp(
+    expected: Vec<u32>,
+    actual: Vec<u32>,
+    epsilon: f64,
+    per_user_credit_cap: u32,
+    dp_mechanism: DpMechanism,
+) {
```

Comment:
per our offline conversation, the `table` below (line 96) is what get's printed out. one idea would be to add two columns, e.g. `Expected Lower Bound (p95)` and `Expected Upper Bound (p95)`. otherwise I think this is now covered all the way up the stack.

Reply:
may take me a little longer to get a good confidence interval as need to analyze sum of truncated discrete laplace.   i'd say we should go ahead and merge these two PRs so we can start observing the performance changes 
1.https://github.com/private-attribution/ipa/pull/1247#pullrequestreview-2283787281
2. https://github.com/private-attribution/ipa/pull/1241

Reply:
makes sense!